notebooks/modeling_a_simple_Einstein_ring.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<p>can you make the residual reduced residuals?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/JAXtronomy/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>